config/testnet/config.ts

```diff
@@ -23,43 +24,27 @@ const actors: any = {
 }
 
 const provingMechanisms: any = {
-  // self: 0, // Destination is Self
-  // settlement: 10, // Source Chain is an L2, Destination is A L1 Settlement Chain
-  settlementL3: 11, // Source Chain is an L3, Destination is a L2 Settlement Chain
-  // bedrock: 20, // Source Chain is an L2, Destination Chain is an L2 using Bedrock
-  // bedrockL2L3: 21, // Source Chain is an L2, Destination Chain is an L3 using Bedrock
-  bedrockL3L2: 22, // Source Chain is an L3, Destination Chain is an L2 using Bedrock
-  // bedrockL1Settlement: 23, // Source Chain is an L1, settlement chain for the Destination Chain which is an L2 using Bedrock
-  bedrockL2Settlement: 24, // Source Chain is the L2, settlement chain for the Destination Chain which is an L3 using Bedrock
-  cannon: 30, // Source Chain is an L2, Destination Chain is an L2 using Cannon
-  cannonL2L3: 31, // Source Chain is an L2, Destination Chain is an L3 using Cannon
-  cannonL3L2: 32, // Source Chain is an L3, Destination Chain is an L2 using Cannon
-  // cannonL1Settlement: 33, // Source Chain is an L1 settlement chain for the Destination Chain which is an L2 using Cannon
-  // cannonL2Settlement: 34, // Source Chain is the L2 settlement chain for the Destination Chain which is an L3 using Cannon
-  hyperProver: 40, // Source Chain is an L2 Destination Chain is an L2 using HyperProver
-  // 0: 'self',
-  // 10: 'settlement',
-  11: 'settlementL3',
-  // 20: 'bedrock',
-  // 21: 'bedrockL2L3',
-  22: 'bedrockL3L2',
-  // 23: 'bedrockL1Settlement',
-  24: 'bedrockL2Settlement',
-  30: 'cannon',
-  31: 'cannonL2L3',
-  // 32: 'cannonL3L2',
-  // 33: 'cannonL1Settlement',
-  // 34: 'cannonL2Settlement',
-  40: 'hyperProver',
+  Self: 0, // Destination is Self
```

Comment:
not sure if these descriptions are all correct - wouldnt hyperprover be the proving type if the destination chain was an L3 using the hyperprover? or if the sourcechain was an L3?

Reply:
may be just that one